[2023-07-30 15:49:06,903] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: read_data_postgres.read_data_postgres manual__2023-07-30T15:49:05.769527+00:00 [queued]>
[2023-07-30 15:49:06,911] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: read_data_postgres.read_data_postgres manual__2023-07-30T15:49:05.769527+00:00 [queued]>
[2023-07-30 15:49:06,912] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 15:49:06,912] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-07-30 15:49:06,912] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 15:49:06,924] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): read_data_postgres> on 2023-07-30 15:49:05.769527+00:00
[2023-07-30 15:49:06,929] {standard_task_runner.py:52} INFO - Started process 714 to run task
[2023-07-30 15:49:06,931] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'read_data_postgres', 'read_data_postgres', 'manual__2023-07-30T15:49:05.769527+00:00', '--job-id', '333', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag_connection.py', '--cfg-path', '/tmp/tmp8967i13d', '--error-file', '/tmp/tmpfdj49cpn']
[2023-07-30 15:49:06,932] {standard_task_runner.py:80} INFO - Job 333: Subtask read_data_postgres
[2023-07-30 15:49:07,013] {task_command.py:369} INFO - Running <TaskInstance: read_data_postgres.read_data_postgres manual__2023-07-30T15:49:05.769527+00:00 [running]> on host 0981bc456e01
[2023-07-30 15:49:07,088] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=read_data_postgres
AIRFLOW_CTX_TASK_ID=read_data_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-07-30T15:49:05.769527+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-30T15:49:05.769527+00:00
[2023-07-30 15:49:07,096] {base.py:68} INFO - Using connection ID 'pg_postgres' for task execution.
[2023-07-30 15:49:07,106] {dbapi.py:208} INFO - Running statement: SELECT * FROM dim_case, parameters: None
[2023-07-30 15:49:07,107] {dbapi.py:216} INFO - Rows affected: 11
[2023-07-30 15:49:07,119] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=read_data_postgres, task_id=read_data_postgres, execution_date=20230730T154905, start_date=20230730T154906, end_date=20230730T154907
[2023-07-30 15:49:07,144] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-30 15:49:07,235] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
